Selenium.Axe/Selenium.Axe.Test/IntegrationTests.cs

```diff
@@ -170,6 +171,18 @@ public void ReportSampleResults(string browser)
             Assert.IsTrue(reportContext.Contains($"Using: {results.TestEngineName} ({results.TestEngineVersion}"));
         }
 
+        [TestMethod]
+        [DataRow("Chrome")]
+        public void RunSiteThatReturnsMultipleTargets(string browser)
+        {
+            var filename = new Uri(Path.GetFullPath(IntegrationTestTargetComplexTargetsFile)).AbsolutePath;
+            InitDriver(browser);
+            _webDriver.Navigate().GoToUrl(filename);
+            new AxeBuilder(_webDriver)
+                .WithOutputFile(@".\raw-axe-results.json")
+                .Analyze();
```

Comment:
This test doesn't contain any assertions. It would be good if it could contain a test that it's successfully parsing the complex bit that the test file was included for (ie, that it successfully retrieves a `['a', ['b', 'c']]` form of `Target`); that would have caught the issue in the 2nd list item of my comment further down in the integration test file

Reply:
Fair point - this now verified that the `Selectors` is set.